draft-ietf-quic-multipath.md

```diff
@@ -575,6 +575,15 @@ any of which it can use when sending packets on that path; see also {{Section 5.
 Usually, it is desired to provide at least one additional connection ID for
 all used paths, to allow for migration.
```

Comment:
I think there are some additional points here that could be clarified:
```suggestion
all used paths, to allow for migration. An endpoint can provide up
to active_connection_ids connection IDs per Path ID
as discussed in {{Section 5.1.1 of QUIC- TRANSPORT}}.
As further specified in {{Section 5.1 of QUIC- TRANSPORT}} connection IDs
cannot be issued more than once on the same connection
and therefore are unique for the scope of the connection,
regardless of the associated Path ID.
```

Reply:
Do we not already state the CID limit per path ID this before? Also, the TP is active_connection_id_limit and it seems there is a space in QUIC- TRANSPORT.
I am not sure we need the last sentence (but we can include it if it really helps).